plugins/woocommerce/client/blocks/tests/js/jest.config.json

```diff
@@ -9,6 +9,7 @@
 	"moduleDirectories": [ "node_modules" ],
 	"moduleNameMapper": {
 		"@wordpress/private-apis": "<rootDir>/node_modules/@wordpress/private-apis",
+		"@wordpress/core-data/build/(.*)$": "<rootDir>/node_modules/@wordpress/core-data/build/$1",
```

Comment:
What this line does? 🤔

Reply:
It's added to support this import:
https://github.com/woocommerce/woocommerce/blob/146d95902fa9b131f5970e0ddb1ee6b36c1d7aa6/packages/js/data/src/setting-options/actions.ts#L10